### ISSUE TITLE
Warning: Could be detected.

### ISSUE THREAD

**Issue by deprale** [danielkrupinski/Osiris#502]
I don't know what got me detected.
I have a private fork of this stuff, and I edited lots of stuff that shoulda changed the signature 100%, the only one I shared my .dll with, is a friend.

WARNING: DON'T SHARE YOUR UNIQUE .DLL you'll get detected.

Used Manual Map Injection. My friend got banned the instant he injected.

**Comment by HowToMakeFaceitAB** [comment]
Use VMprotect or themida cracked

**Comment by deprale** [comment]
U cant VMProtect if u want manualmap injection.

Shouldnt have been an issue either way since it had an unique signature.

**Comment by deprale** [comment]
Disconnect: Kicked by Console : Account is Untrusted.
Disconnect: Kicked by Console : Account is Untrusted.

Another acc :D

**Comment by hydemods** [comment]
im using the public fork and a public injector. havent gotten vacced or utd

**Comment by deprale** [comment]
Just got another untrusted.

**Comment by Floraly** [comment]
And I thaught there is something wrong on my Computer. All my Accounts got VAC Ban. I only used Osiris and a private Injector. I use the Injector also for my Teamfortress and CS:Source Cheats. :(

**Comment by deprale** [comment]
Another untrusted.

Seems like something's wrong when using the aimbot but I can't confirm, I raged on two new accs on ffa dm community. 26-34 minutes first and second accs untrusted, third same ban, fourth I sit afk 30 minutes, no untrusted, use bhop 100 hitchance  for 15 minutes straight, then afk for another 15 while have space bugged (space +shift+tab = infinite key stuck), no ban, as soon as i try turning on aimbot to rage a bit (fov 10, 1 smooth, recoil control x/y both on 1, shot/aim 0.50 mouse1 toggle) = insta ban. Doesn't matter if i use silent aim or not.

i don't know if its exactly the aimbot :thonking: might be just my private fork but I've only added Viewmodel X/Y/Z changer + left handed knife feature. Fucking strange... I got banned on my "main" prime acc while not using the aimbot at all sooo idk what to say, maybe it really got detected or some shit.
I already used different injectors all vmprotected, tried different methods, manual mapping, load library... native inject, every time the same.

**Comment by deprale** [comment]
Yeah, dont use manual mapping :)

**Comment by Shaxzy** [comment]
Weird.. I have been using this for 6 months, NEVER got any vac ban using this. I have changed a ton of things + using selfmade injector. I can’t tell what’s causing the ban for you, but usually untrusted bans are caused by detected injectors and/or un-clamped anti-aim.

**Comment by 0xE232FE** [comment]
Omg 2 Accounts that got VAC Banned have now a 2nd Ban Overwatch infraction. I did not rage with that Accounts and do not use Occluded Chams or any kind of wallhack. I only legit hacking. With Healthbased Chams and i only see them when they are visible.

What does this means?
VAC Ban first then Game Ban by Overwatch?
Injector detected? I used Inflame.asm injector.

**Comment by HowToMakeFaceitAB** [comment]
Probably detectes injector i use private inn never detected

**Comment by 0xE232FE** [comment]
Okay I just tested Inflames.asm and two test Accounts got first VAC Ban and then 3-5 Days later they got an Additional Game Ban. With one Account I only stay AFK on my local VAC Protected server. Omg it's nice to know its the injector. Daniel already said he will update inflames but I use actually my own injector which seems don't have problems.
I use same Email Address on all my Accounts, they were not affected and still clean no VAC or Game Ban. But I noticed more Rage Cheaters in my Game. Seems VAC Net is doing its job. XD But after a week of legit cheating the Rage Cheaters getting lesser and lesser again, so my Trustfactor seems to rise again. :D
